Formula/c/certifi.rb

```diff
@@ -15,24 +15,37 @@ class Certifi < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "1f1fc985a1c89bd40c73b17e3dfbf5483cb0417c8d4d12e2be66158a503ab169"
   end
 
-  depends_on "python@3.11" => [:build, :test]
-  depends_on "python@3.12" => [:build, :test]
   depends_on "python@3.13" => [:build, :test]
+  depends_on "python@3.11" => :test
+  depends_on "python@3.12" => :test
```

Comment:
In terms of support range, considering 2 options:
* Minimum 3.11 since we have been providing that so far
* Minimum 3.12 since we only started setting `EXTERNALLY-MANAGED` here